tools/utils/parse-stack.ts

```diff
@@ -110,28 +137,31 @@ function parseStackFrames(frames) {
         // m[1] could be Object.__top_mark__ or something like that
         // depending on where exactly you put the function returned by
         // markTop
-        ret = [];
+        parsedFrames = [];
         return;
       }
+
       if (m[1].match(/(?:^|\.)__bottom_mark__$/)) {
         return stop = true;
       }
-      ret.push({
+
+      parsedFrames.push({
         func: m[1],
         file: m[5],
-        line: m[7] ? +m[7] : undefined,
-        column: m[9] ? +m[9] : undefined
+        line: m[7] ? +m[7] : null,
```

Comment:
🆗 I wondered if these changes from `undefined` to `null` would be safe, and it looks ok, because the checks in `buildmessage.js` are for thruthiness:

https://github.com/meteor/meteor/blob/5c32331591f268f2acdac5a1a7481eb9e65dc3d4/tools/utils/buildmessage.js#L68-L75

https://github.com/meteor/meteor/blob/5c32331591f268f2acdac5a1a7481eb9e65dc3d4/tools/utils/buildmessage.js#L83-L85

Reply:
thanks for double-checking and confirming!

Reply:
I like to manually use `null` when I'm marking something for falsyness while otherwise leaving `undefined` as a feature of the language that tells me when something is _actually undefined_ and never manually using `undefined` if I can avoid it. It's a semantic difference.

Reply:
Yes, I feel the same way.